libcontainer/cgroups/fs/fs.go

```diff
@@ -69,6 +61,12 @@ func NewManager(cg *configs.Cgroup, paths map[string]string) (cgroups.Manager, e
 		return nil, cgroups.ErrV1NoUnified
 	}
 
+	// If using cgroups-hybrid mode then add a "" controller indicating
+	// it should join the cgroups v2.
+	if cgroups.IsCgroup2HybridMode() {
+		subsystems = append(subsystems, &NameGroup{GroupName: "", Join: true})
```

Comment:
This is mutating a package var in a constructor. Can that race with other readers of subsystems? If the constructor is called multiple times, this will append multiple times, right?

Reply:
Ugh, yeah, from a quick look, we could move the subsystems under the manager instance.

Reply:
Pushed a commit to test that out (will check later).